client/src/components/screens/Signup.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Redirect } from 'react-router-dom'
+
+import { clearAuthErrorMsg } from '../../actions'
+
+import Page from '../Page'
+import SignUpForm from '../forms/SignUpForm'
+
+import PageContent from '../styles/PageContentStyles'
+import Headline from '../styles/PageHeadline'
+import ErrorMsg from '../styles/ErrorMsg'
+
+class Signup extends Component {
+  componentWillUnmount() {
+    if (this.props.auth.errorMsg) {
+      this.props.clearAuthErrorMsg()
+    }
+  }
+
+  render() {
+    const { auth } = this.props
+
+    if (auth.authenticated) return <Redirect to="/" />
```

Comment:
I strongly recommend to use curly braces {} to wrap every if-elseif-else statement/s. Regardeles, single or multiple lines.

Reply:
Fixed!